crates/router/src/core/fraud_check.rs

```diff
@@ -170,6 +170,14 @@ where
                     id: merchant_account.get_id().get_string_repr().to_owned(),
                 })
                 .ok();
+            let enabled_merchant_connector_account_from_db_option =
```

Comment:
add appropriate logs saying that the mca is disabled. Also add logs in the previous step where we are fetching the merchant account and then ignoring the error.